<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasGen</class>
 <widget class="QWidget" name="PasGen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="contentWidget" native="true">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>600</width>
     <height>600</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>567</width>
     <height>532</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
	background-color: white;
}
QCheckBox::indicator:unchecked{
	border-radius: 4px;
	background-color: #bbbbbb;
}
QCheckBox::indicator:unchecked:hover{
	background-color: #aaaaaa;
}
QCheckBox:unchecked {
	color: #555555;
}
QCheckBox:unchecked:hover {
	color: #444444;
}
QCheckBox:checked {
	color: black;
}
QSlider{
	selection-background-color: #bf3636;
}
#counter {
	color: black;
	font-weight: bold;
	font-size: 14px;
}
#addBtn
{
	background: #d43f3f;
	border: none;
	border-radius: 5px;
}
#addBtn:hover
{
	background: #f04848;
}
#pasGenEdit {
	color: black;
	font-size: 16px;
	padding-left: 10px;
}</string>
   </property>
   <widget class="QLabel" name="pageTitle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black; font-size: 20px; font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Password Generator</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pasGenEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>561</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>64</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Normal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="capitalRad">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="text">
     <string>Capital Letters</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="numberRad">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="text">
     <string>Numbers</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="charRad">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="text">
     <string>Special Characters</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="smallRad">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="text">
     <string>Small Letters</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="charCountSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>521</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>32</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="counter">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>130</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>561</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>212</red>
          <green>63</green>
          <blue>63</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>212</red>
          <green>63</green>
          <blue>63</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>212</red>
          <green>63</green>
          <blue>63</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>212</red>
          <green>63</green>
          <blue>63</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>212</red>
          <green>63</green>
          <blue>63</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>212</red>
          <green>63</green>
          <blue>63</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>212</red>
          <green>63</green>
          <blue>63</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>212</red>
          <green>63</green>
          <blue>63</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>212</red>
          <green>63</green>
          <blue>63</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::ClickFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Generate New</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="menuWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>600</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>500</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>200</width>
     <height>532</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>191</red>
         <green>54</green>
         <blue>54</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>191</red>
         <green>54</green>
         <blue>54</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>191</red>
         <green>54</green>
         <blue>54</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>191</red>
         <green>54</green>
         <blue>54</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>191</red>
         <green>54</green>
         <blue>54</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>191</red>
         <green>54</green>
         <blue>54</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>191</red>
         <green>54</green>
         <blue>54</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>191</red>
         <green>54</green>
         <blue>54</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>191</red>
         <green>54</green>
         <blue>54</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font-size: 16px;
	font-weight: bold;
	background-color: #bf3636;
	color: white;
	padding: 8px;
	padding-left: 24px;
	border: none;
	border-left: 2px solid white;
	text-align:left;
}
QPushButton:hover {
	background-color: #d43f3f;
}
QWidget{
	background-color: #bf3636;
}</string>
   </property>
   <widget class="QPushButton" name="pasListBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Password List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pasGenBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-left: 2px solid #cccccc;</string>
    </property>
    <property name="text">
     <string>Password Generator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pasCheckBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="text">
     <string>Check Strength</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logOutBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/res/pasman.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
